### PR TITLE
Add Pub package versions to VersionResponse

### PR DESCRIPTION
**Pull request by srawlins**
Deploy with https://github.com/dart-lang/dart-pad/pull/1887.

Simple change adds Pub package versions to VersionResponse, and fills in data with appropriate pubspec.lock info.

### CODE REVIEW

File: cloud_run.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM google/dart:2.12.4
+FROM google/dart:2.12.2
```
Comment (domesticmouse) [inline cloud_run.Dockerfile:1]: We should think about removing the docker file update from the grind script.

File: lib/src/common_server_impl.dart [hunk 2]
```diff
@@ -151,9 +152,11 @@ class CommonServerImpl {
               <String, String>{});
   }
 
-  Future<proto.VersionResponse> version(proto.VersionRequest _) {
+  Future<proto.VersionResponse> version(proto.VersionRequest request) {
```
Comment (domesticmouse) [inline lib/src/common_server_impl.dart:155]: Is the `request` used?

**Review by domesticmouse — APPROVED** [review]

File: cloud_run.Dockerfile (hunk 1 above)
Comment (domesticmouse) [reply cloud_run.Dockerfile:1]: (revert this change)

File: lib/src/common_server_impl.dart (hunk 2 above)
Comment (srawlins) [reply lib/src/common_server_impl.dart]: Good catch, reverted.

File: cloud_run.Dockerfile (hunk 1 above)
Comment (srawlins) [reply cloud_run.Dockerfile]: done

**Pull request merged**